liwords-ui/src/gameroom/notepad.tsx

```diff
@@ -135,6 +135,30 @@ export const Notepad = React.memo((props: NotepadProps) => {
     }
     numWolgesWas.current = numWolges;
   }, [numWolges]);
+  const controls = useCallback(
```

Comment:
- how about `useMemo` (and use `controls` instead of `controls()` at the two places below)?
- would it be desirable to also set focus on the emptied notepad (`notepadEl.current?.focus();`)?